lib/irb/debug.rb

```diff
@@ -60,7 +60,7 @@ def DEBUGGER__.capture_frames(*args)
         if !DEBUGGER__::CONFIG[:no_hint] && irb.context.io.is_a?(RelineInputMethod)
           Reline.output_modifier_proc = proc do |input, complete:|
             unless input.strip.empty?
-              cmd = input.split(/\s/, 2).first
+              cmd = input[/\S+/]
```

Comment:
The behavior is changed (example input: `'  info  '`) but the changed behavior matches better to debug command recognization rule. :+1: